jira_view.py

```diff
@@ -13,7 +13,7 @@
 # either express or implied. See the License for the specific language governing permissions
 # and limitations under the License.
 def get_ctx_result(provides, result):
-    """ Function that parses data.
+    """Function that parses data.
```

Comment:
Can we make this descriptions more descriptive? Probably many functions in programming world parse some data

Reply:
Done